src/utils/UserUtils.ts

```diff
@@ -4,20 +4,28 @@ export const getUserType = (user: User) => {
   if (user.type === 'pupil') {
     return `Schüler*in`;
   }
+  return `Student*in`;
+};
+
+export const getUserTags = (user: User) => {
+  const studentTags = {};
+  if (user.type === 'pupil') {
+    return { pupil: 'Schüler*in' };
+  }
 
   if (user.isTutor) {
-    return `Student*in`;
+    studentTags["isTutor"] = 'Tutor';
   }
 
   if (user.isInstructor) {
-    return 'Tutor*in';
+    studentTags["isInstructor"] =
+      user.university === null ? 'Kursleiter' : 'Praktikant';
   }
 
   if (user.isProjectCoach) {
-    return 'Project-Coach';
+    studentTags["isProjectCoach"] = 'Project-Coach';
```

Comment:
And to be fully consistent:  it's `Projektcoach*in` here as well

Reply:
Additionally, typing `studentTags` and writing `studentTags.isProjectCoach = ` would be preferred and would fix eslint. This also applies for the other names above.

Reply:
so gero i can't understand exactly what you mean with that,
but generally we tried to idetify studentTags as new Object(), and hence could use the dot operator like that "studentTags.isProjectCoach = " and the code worked just fine, but then we couldn't really push it for some reason, if this error is relevent, i can try to replicate again, and take a screen shot of it.